src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+                "      ( ==  ^  == )\n" +
+                "       )         (\n" +
+                "      (           )\n" +
+                "     ( (  )   (  ) )\n" +
+                "    (__(__)___(__)__)";
+        System.out.println(catLogo);
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm NEKOBOT!!");
+        System.out.println("    What can I do for you :>");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Prints a goodbye message.
+     */
+    public static void goodbyeMessage() {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Bye~ Hope to see you again soon ;w;");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Prints all tasks in the TaskList.
+     * @param taskList An ArrayList containing all existing Tasks.
+     */
+    public static void listMessage(ArrayList<Task> taskList) {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Here are the tasks in your list!!");
+        int index = 1;
+        for (Task task : taskList) {
+            System.out.println("    " + index + ". " + task);
+            index++;
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Prints all tasks whose date is set as the current date (today).
+     * @param taskList An ArrayList containing all existing Tasks.
+     */
+    public static void todayMessage(ArrayList<Task> taskList) {
+        // TODO: refactor to filter in duke.task.TaskList class
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Here are today's tasks!!");
+        int index = 1;
+        for (Task task : taskList) {
+            if (task.isToday()) {
+                System.out.println("    " + index + ". " + task);
+                index++;
+            }
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public static void findMessage(ArrayList<Task> taskList) {
+        if (taskList.size() == 0) {
+            System.out.println("    ____________________________________________________________");
+            System.out.println("    Oh dear, I couldn't find any matching tasks :o");
+            System.out.println("    ____________________________________________________________");
+        } else {
+            System.out.println("    ____________________________________________________________");
+            System.out.println("    Here are your search results!!");
+            int index = 1;
+            for(Task task: taskList) {
```

Comment:
Coding standard asked for a space after for. So I think it should be: for (Task task: taskList) {